src/config.ts

```diff
@@ -5,6 +5,11 @@ export const portDefault = 9000
 export interface EmberPlusConfig {
   host?: string
   port?: number
+  take?: boolean
+  selectedSource: number[]
+  selectedDestination: number[]
+  matrices?: string[]
```

Comment:
It feels like these should not be part of the config data as they are changed at runtime and not instantiated until the model is loaded.

Instead I think they should be kept as a structure on the main module class and passed to the action functions as payload.